doc/build/zephyr_cmake_package.rst

```diff
@@ -313,44 +313,70 @@ Such a CMakeLists.txt could look as:
 
 .. _cmake_build_config_package:
 
-Zephyr Build Configuration CMake package
-****************************************
+Zephyr Build Configuration CMake packages
+*****************************************
 
-The Zephyr Build Configuration CMake package provides a possibility for a Zephyr based project to
-control Zephyr build settings in a generic way.
+There are two Zephyr Build configuration packages which provide control over the build
+settings in Zephyr in a more generic way. These packages are:
 
-It is similar to the per-user ``.zephyrrc`` file that can be used to set :ref:`env_vars`, but it
-sets CMake variables instead. It also allows you to automatically share the build configuration
-among all users through the project repository. It also allows more advanced use cases, such as loading
-of additional CMake boilerplate code.
+* **ZephyrBuildConfiguration**: Applies to all Zephyr applications in the workspace
+* **ZephyrAppConfiguration**: Applies only to the application you are currently building
 
-The Zephyr Build Configuration CMake package will be loaded in the Zephyr boilerplate code after
-initial properties and ``ZEPHYR_BASE`` has been defined, but before CMake code execution.
-This allows the Zephyr Build Configuration CMake package to setup or extend properties such as:
+They are similar to the per-user ``.zephyrrc`` file that can be used to set :ref:`env_vars`, but
+they set CMake variables instead. They also allow you to automatically share the build configuration
+among all users through the project repository. They also allow more advanced use cases, such as
+loading of additional CMake boilerplate code.
+
+The Zephyr Build Configuration CMake packages will be loaded in the Zephyr boilerplate code after
+initial properties and ``ZEPHYR_BASE`` has been defined, but before CMake code execution. The
+ZephyrBuildConfiguration is included first and ZephyrAppConfiguration afterwards. That means the
+application-specific package could override the workspace settings, if needed.
+This allows the Zephyr Build Configuration CMake packages to setup or extend properties such as:
 ``DTS_ROOT``, ``BOARD_ROOT``, ``TOOLCHAIN_ROOT`` / other toolchain setup, fixed overlays, and any
 other property that can be controlled. It also allows inclusion of additional boilerplate code.
 
-To provide a Zephyr Build Configuration CMake package, create ``ZephyrBuildConfig.cmake`` and place
-it in a Zephyr workspace top-level folder as:
+To provide a ZephyrBuildConfiguration or ZephyrAppConfiguration, create ``ZephyrBuildConfig.cmake``
+and/or ``ZephyrAppConfig.cmake`` respectively and place them in the appropriate location. The CMake
```

Comment:
use
```
:file:`<name>`
```
syntax for files

Reply:
As a side note: In the documentation guidelines, there is still this sentence: 

> You can also use the \`\`inline code\`\` markup (double backticks) to indicate a `filename`.

Maybe this should be changed or clarified.